src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -34,29 +35,41 @@
         </v-card-text>
       </template>
 
-      <!--  List Tables -->
-      <template v-else-if="!showSocietiesInfo">
-        <v-card-text class="">
+      <!-- Special case for societies -->
```

Comment:
I moved this up so that the code if "if societies" instead of "if not societies".